packages/expo-system-ui/ios/ExpoSystemUI/ExpoSystemUIModule.swift

```diff
@@ -3,22 +3,56 @@
 import ExpoModulesCore
 
 public class ExpoSystemUIModule: Module {
+  
+  public required init(appContext: AppContext) {
```

Comment:
You can probably do this in `onCreate` definition component 😉 It's called a bit later, but I think it doesn't matter in this case.